Sources/Authenticator/Options/TOTPOptions.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Options for configuring the TOTP MFA Experience
+public struct TOTPOptions {
+
+    /// The `issuer` is the title displayed in a user's TOTP App preceding the
+    /// account name. In most cases, this should be the name of your app.
+    /// For example, if your app is called "My App", your user will see
+    /// "My App" - "username" in their TOTP app.
+    /// Defaults to the `appName` found in the project configuration.
+    public let issuer: String?
```

Comment:
This property does not really default to the `appName`, since that logic is outside this structure.

Since you absolutely need to have a non-nil value when you use it in `ContinueSignInWithTOTPSetupView`, I'd prefer for `issuer` to be a non-nil String.

Since `totpOptions` can be nil anyway in the Authenticator's `init`, then you can still have the fallback logic to use the appName